hex/sd_api_v3/sdk150_connectivity_743b4d0.patch

```diff
@@ -13,3 +13,20 @@
      .rfu1               = 0xFF,
      .sd_ble_api_version = NRF_SD_BLE_API_VERSION,
      .transport_type     = 1,
+--- a/components/serialization/connectivity/ser_conn_reset_cmd_decoder.c
++++ b/components/serialization/connectivity/ser_conn_reset_cmd_decoder.c
+@@ -7,7 +7,6 @@
+ #include "ble_serialization.h"
+ #include "nrf_log.h"
+ #include "ser_conn_handlers.h"
+-#include "ser_hal_transport.h"
+ static bool m_reset_ongoing;
```

Comment:
I might be wrong - reading patch files are tricky - but it seems there are probably more files included than used, and `m_reset_ongoing` is not initialized or used.

Reply:
The only thing this patch does is to remove the ser_hal_transport.h include, and remove the ser_hal_transport_reset() function call. Reviewing the surrounding code would be to review the SDK implementation, which can be considered out-of-scope of this review :-)